react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,43 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
-  }
+  renderSquare = (index, value) => <Square value={value} onClick={() => this.props.handleClick(index)} />;
 
   render() {
-    const status = 'Next player: X';
+    const { squares, status } = this.props;
+    const { boardRow } = styles;
     return (
-      <div>
+      <Fragment>
         <div className={styles.status}>{status}</div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(0)}
-          {this.renderSquare(1)}
-          {this.renderSquare(2)}
+        <div className={boardRow}>
```

Comment:
boardRow doesnt exist. I think this should be  `styles.boardRow`

Also, notice that you are repeating this row structure three times. Maybe you could extract to define this structure just once, and reuse it.